EdgeDeflector/resources/nsis_installer.nsi

```diff
@@ -1,72 +1,237 @@
+  !insertmacro MUI_LANGUAGE "Belarusian"
+  !insertmacro MUI_LANGUAGE "Icelandic"
+  !insertmacro MUI_LANGUAGE "Malay"
+  !insertmacro MUI_LANGUAGE "Bosnian"
+  !insertmacro MUI_LANGUAGE "Kurdish"
+  !insertmacro MUI_LANGUAGE "Irish"
+  !insertmacro MUI_LANGUAGE "Uzbek"
+  !insertmacro MUI_LANGUAGE "Galician"
+  !insertmacro MUI_LANGUAGE "Afrikaans"
+  !insertmacro MUI_LANGUAGE "Catalan"
+  !insertmacro MUI_LANGUAGE "Esperanto"
+  !insertmacro MUI_LANGUAGE "Asturian"
+  !insertmacro MUI_LANGUAGE "Basque"
+  !insertmacro MUI_LANGUAGE "Pashto"
+  !insertmacro MUI_LANGUAGE "Georgian"
+  !insertmacro MUI_LANGUAGE "Vietnamese"
+  !insertmacro MUI_LANGUAGE "Welsh"
+  !insertmacro MUI_LANGUAGE "Armenian"
+  !insertmacro MUI_LANGUAGE "Corsican"
 
 ;--------------------------------
+;Before Installation Function
+
+Function .onInit
 
+  ;Choose the language pop-up at the start
+  !insertmacro MUI_LANGDLL_DISPLAY
 
-Section "Installer"
+FunctionEnd
+
+;--------------------------------
+;Installer Sections
 
-  ; Set output path to the installation directory.
+Section "EdgeDeflector (Required)"
+  SectionIn RO # Just means if in component mode this is locked
+
+  ;Set output path to the installation directory.
   SetOutPath $INSTDIR
 
-  ; Put file there
-  File ..\bin\Release\EdgeDeflector.exe
+  ;Put file here
+  File "..\bin\Release\${PRODUCT}.exe"
 
-  ; Self-registeres to the registry
+  ;Execute EdgeDeflector.exe for self-registering
   ExecWait '$INSTDIR\${PRODUCT}.exe'
 
-  ; Write the installation path to the registry
-  WriteRegStr HKLM SOFTWARE\${PRODUCT} "Install_Dir" "$INSTDIR"
-  
-  ; Write the uninstall keys to the registry
+  ;Store installation folder in registry
+  WriteRegStr HKLM "Software\${PRODUCT}" "" $INSTDIR
+
+  ;Registry information for add/remove programs
   WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "DisplayName" "${PRODUCT}"
   WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "UninstallString" '"$INSTDIR\${PRODUCT}_uninstall.exe"'
+  WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "URLInfoAbout" "$\"${ABOUTURL}$\""
+  WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "DisplayVersion" "$\"${VERSIONMAJOR}.${VERSIONMINOR}.${VERSIONBUILD}$\""
+  WriteRegDWORD HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "VersionMajor" ${VERSIONMAJOR}
+  WriteRegDWORD HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "VersionMinor" ${VERSIONMINOR}
   WriteRegDWORD HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "NoModify" 1
   WriteRegDWORD HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "NoRepair" 1
-  WriteUninstaller "${PRODUCT}_uninstall.exe"
 
-  ExecShell "open" "microsoft-edge:https://github.com/da2x/EdgeDeflector/wiki/Thank-you-for-installing-EdgeDeflector"
+  ;Create start-menu items (shortcuts)
```

Comment:
I’ve no intention of adding junk (the uninstaller) in people’s Start menu. That is entirely pointless. Windows has two places users go to uninstall any program, and including it in the Start menu also would just be clutter. I’d [never add a folder to the Start menu](https://ctrl.blog/entry/windows-start-menu-folders) anyway. The practice is a left-over from the very early days of Windows.

Reply:
I'll added it because I still use it (also because of if you input `EdgeD` in Cortana you will instantly see/find the uninstaller).
But if you don't like it I can remove it.